src/utils/businessHours.test.ts

```diff
@@ -329,7 +329,7 @@ describe('businessHours tests', function () {
     it('should calculate SLO violation if label is waiting for product owner', async function () {
       const result = await calculateSLOViolationTriage(
         WAITING_FOR_PRODUCT_OWNER_LABEL,
-        [{ name: 'Test' }],
```

Comment:
Can we have a test somewhere that fails when name: `Test` like it was before, to add the fact that the prefix is required to the spec?